apps/rush-lib/src/logic/setup/ArtifactoryConfiguration.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { JsonFile, JsonSchema, FileSystem } from '@rushstack/node-core-library';
+
+export interface IArtifactoryPackageRegistryJson {
+  enabled: boolean;
+  globallyMappedNpmScopes?: string[];
```

Comment:
What if you want everything to go through your private feed?

Reply:
I was going to design that later, but now that I think about it, the name ` globallyMappedNpmScopes` will need to make sense with it.  I'll include it in this PR.

Reply:
I replaced `globallyMappedNpmScopes` with a more versatile `userNpmrcLinesToAdd` that is just free-form strings to be injected into `.npmrc`.  The merge algorithm is very careful not to duplicate these lines when `rush setup` is run repeatedly.